src/sample/conference/src/main/res/layout/fragment_video.xml

```diff
@@ -39,16 +39,18 @@
         </LinearLayout>
 
 
-        <org.webrtc.SurfaceViewRenderer
-            android:id="@+id/small_renderer"
-            android:layout_width="160dp"
-            android:layout_height="120dp"/>
+
 
         <org.webrtc.SurfaceViewRenderer
             android:id="@+id/full_renderer"
             android:layout_width="match_parent"
             android:layout_height="match_parent"/>
 
+        <org.webrtc.SurfaceViewRenderer
+            android:id="@+id/small_renderer"
+            android:layout_width="160dp"
+            android:layout_height="120dp"/>
+
```

Comment:
Please don't submit changes that are unnecessary.

Reply:
Sometimes the full_renderer will cover the small_renderer if it above small_renderer.So I put it down.
If the small_renderer is useless, I will put it back.